### PR TITLE
Merge: Mixed usage of ORDERLABEL / LABEL on newspaper export (#286)

### LINKED ISSUE #286: Mixed usage of ORDERLABEL / LABEL on newspaper export
On exporting a newspaper the usage of ``ORDERLABEL`` and ``LABEL`` inside logical structmap is mixed. For year ``LABEL`` is used and for month and day ``ORDERLABEL``.

```xml
  <mets:structMap TYPE="LOGICAL">
    <mets:div CONTENTIDS="http://digital.slub-dresden.de/id391878840" ID="LOG_0007" LABEL="Leipziger jüdische Wochenschau" ORDERLABEL="Leipziger jüdische Wochenschau" TYPE="newspaper">
      <mets:mptr xmlns:xlink="http://www.w3.org/1999/xlink" LOCTYPE="URL" xlink:href="http://digital.slub-dresden.de/fileadmin/data/391878840/391878840_anchor.xml"/>
      <mets:div ADMID="AMD" ID="LOG_0008" LABEL="1930" TYPE="year">
        <mets:mptr xmlns:xlink="http://www.w3.org/1999/xlink" LOCTYPE="URL" xlink:href="http://digital.slub-dresden.de/fileadmin/data/391878840/391878840_year.xml"/>
        <mets:div ADMID="AMD" ID="LOG_0009" ORDERLABEL="1" TYPE="month">
          <mets:div ADMID="AMD" ID="LOG_0010" ORDERLABEL="17" TYPE="day">
            <mets:div ADMID="AMD" DMDID="DMDLOG_0004" ID="LOG_0011" TYPE="issue"/>
          </mets:div>
        </mets:div>
      </mets:div>
    </mets:div>
  </mets:structMap>
```

Is it possible to change this to

- both ``ORDERLABEL`` and ``LABEL``
- only ``ORDERLABEL``
- only ``LABEL``

? First solution would be best solution.

### PR DESCRIPTION
**Pull request by matthias-ronge**
Fixes #286

### CODE REVIEW

**Comment by matthias-ronge** [comment]
Fixed

**Comment by henning-gerhardt** [comment]
Works no for me including issue #302 is now fixed.

**Comment by henning-gerhardt** [comment]
@matthias-ronge : I was to fast and I missed a ``LABEL`` / ``ORDERLABEL`` in ``_year`` anchor file for referencing to newspaper:

wrong:
```xml
  <mets:structMap TYPE="LOGICAL">
    <mets:div CONTENTIDS="http://digital.slub-dresden.de/id391878840" ID="LOG_0002" TYPE="newspaper">
      <mets:mptr xmlns:xlink="http://www.w3.org/1999/xlink" LOCTYPE="URL" xlink:href="http://digital.slub-dresden.de/fileadmin/data/391878840/391878840_anchor.xml"/>
      <mets:div ADMID="AMD" DMDID="DMDLOG_0002" ID="LOG_0003" LABEL="19310" ORDERLABEL="1931" TYPE="year">
        <mets:div ADMID="AMD" ID="LOG_0004" ORDERLABEL="1" TYPE="month">
          <mets:div ADMID="AMD" ID="LOG_0005" ORDERLABEL="6" TYPE="day">
            <mets:div ID="LOG_0006" LABEL="10" ORDERLABEL="1" TYPE="issue">
              <mets:mptr xmlns:xlink="http://www.w3.org/1999/xlink" LOCTYPE="URL" link:href="http://digital.slub-dresden.de/fileadmin/data/391878840/391878840.xml"/>
            </mets:div>
          </mets:div>
        </mets:div>
      </mets:div>
    </mets:div>
  </mets:structMap>
```

correct:
```xml
  <mets:structMap TYPE="LOGICAL">
    <mets:div CONTENTIDS="http://digital.slub-dresden.de/id391878840" ID="LOG_0002" LABEL="Leipziger jüdische Wochenschau" TYPE="newspaper">
      <mets:mptr xmlns:xlink="http://www.w3.org/1999/xlink" LOCTYPE="URL" xlink:href="http://digital.slub-dresden.de/fileadmin/data/391878840/391878840_anchor.xml"/>
      <mets:div ADMID="AMD" DMDID="DMDLOG_0002" ID="LOG_0003" LABEL="19310" ORDERLABEL="1931" TYPE="year">
        <mets:div ADMID="AMD" ID="LOG_0004" ORDERLABEL="1" TYPE="month">
          <mets:div ADMID="AMD" ID="LOG_0005" ORDERLABEL="6" TYPE="day">
            <mets:div ID="LOG_0006" LABEL="10" ORDERLABEL="1" TYPE="issue">
              <mets:mptr xmlns:xlink="http://www.w3.org/1999/xlink" LOCTYPE="URL" xlink:href="http://digital.slub-dresden.de/fileadmin/data/391878840/391878840.xml"/>
            </mets:div>
          </mets:div>
        </mets:div>
      </mets:div>
    </mets:div>
  </mets:structMap>
```

**Comment by matthias-ronge** [comment]
The missing `x` (`link` instead of `xlink`) in your *wrong* example is a typo, isn’t it?

      <mets:structMap TYPE="LOGICAL">
        <mets:div CONTENTIDS="http://digital.slub-dresden.de/id391878840" ID="LOG_0002" TYPE="n…
          <mets:mptr xmlns:xlink="http://www.w3.org/1999/xlink" LOCTYPE="URL" xlink:href="http:…
          <mets:div ADMID="AMD" DMDID="DMDLOG_0002" ID="LOG_0003" LABEL="19310" ORDERLABEL="193…
            <mets:div ADMID="AMD" ID="LOG_0004" ORDERLABEL="1" TYPE="month">
              <mets:div ADMID="AMD" ID="LOG_0005" ORDERLABEL="6" TYPE="day">
                <mets:div ID="LOG_0006" LABEL="10" ORDERLABEL="1" TYPE="issue">
                  <mets:mptr xmlns:xlink="http://www.w3.org/1999/xlink" LOCTYPE="URL" link:href…
                                                                                      ^
                                                                                     here

**Comment by henning-gerhardt** [comment]
Yes, it is a typo - something goes while copy and pasting wrong :/

**Comment by henning-gerhardt** [comment]
As far I can say: its look good for me. @sebastian-meyer will have a look too but maybe first next week.

File: Goobi/src/de/sub/goobi/helper/tasks/INameableTask.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+ * 
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
+ * details.
+ * 
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, write to the Free Software Foundation, Inc., 59 Temple
+ * Place, Suite 330, Boston, MA 02111-1307 USA
+ *
```
Comment (stweil) [inline Goobi/src/de/sub/goobi/helper/tasks/INameableTask.java]: The FSF address is wrong. It changed many years ago.

**Comment by henning-gerhardt** [comment]
After exporting some Goobi processes we found no obviously error more. I will merge this.

**Pull request merged**